libs/single-spa-angular/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "../node_modules/ng-packagr/ng-package.schema.json",
   "name": "single-spa-angular",
-  "version": "5.0.1",
+  "version": "5.0.2",
```

Comment:
Please revert - version numbers are bumped during publish, not in each PR

Reply:
Oh sorry. I updated it.